www/src/pages/TR/2025.10/index.html

```diff
@@ -315,13 +325,13 @@ <h1 id="title" class="title">Design Tokens Technical Reports 2025.10</h1>
 </section>
 
     <section class="appendix" id="acknowledgements"><div class="header-wrapper"><h2 id="a-acknowledgements"><bdi class="secno">A. </bdi>Acknowledgements</h2><a class="self-link" href="#acknowledgements" aria-label="Permalink for Appendix A."></a></div>
-
+      
       <p>
         We'd also like to thank the following contributors:
-        <span id="gh-contributors">Abhishek Warokar, Adam Stankiewicz, Adekunle Oduye, Ale Muñoz, Asher, Benjamin Kindle, Bjørn Madsen, Chase McCoy, Copilot, Dale Sande, Daniel Flynn, Daniel Rinehart, Danny Banks, Dominique Hazael-Massieux, Evan Lovely, Fabian Friedl, Guilherme Nagüeva, Guy Lepage, Ivan Maksimovic, James Nash, Jan Toman, Jina Anne, Jon Levine, Joren Broekema, Kaelig Deloumeau-Prigent, Kevin Powell, Kilian Valkhof, KLS, Laurent Thiebault, Lukas Oppermann, Marcos Castro, Mark Tomlinson, Matt Felten, Matt Ström-Awn, Mike Kamminga, Miriam Suzanne, Namık Özgür Aydın, nicolaibach, Pavel Laptev, Pavel Vostrikov, Red Huang, Rob Eisenberg, Robin, Roman Komarov, Salem Cobalt, Sébastien Barbieri, Sigurd Fosseng, Stuart Robson, Vsevolod Dolgopolov (aka Seva), and Zhihao Cui</span>.
```

Comment:
It’s OK to remove Mike’s name from here; he’s listed as an editor on all relevant documents